docs/resources/integration.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "fastly"
+page_title: "Fastly: integration"
+sidebar_current: "docs-fastly-resource-integration"
+description: |-
+  Provides a Fastly integration. Fastly will use these integrations to send you notifications and alerts related to the Fastly product. Mailing list, Microsoft Teams, New Relic, PagerDuty, Slack, and webhook are supported as integrations.
+---
+
+# fastly_integration
+
+Provides a Fastly integration. Fastly will use these integrations to send you notifications and alerts related to the Fastly product. Mailing list, Microsoft Teams, New Relic, PagerDuty, Slack, and webhook are supported as integrations.
+
+## Example Usage
+
+```terraform
+# IMPORTANT: mailing list integrations require confirmation.
+# To send a confirmation email and verify integration status,
+# after applying changes using Terraform, please visit
+# https://manage.fastly.com/observability/alerts/integrations
```

Comment:
I'm now thinking we should move this to a note in the documentation itself rather than the code example (see my suggested edit for `templates/resources/integration.md.tmpl`)
```suggestion
```